src/OpenTelemetry.Extensions.PersistentStorage/FileBlob.cs

```diff
@@ -97,7 +97,7 @@ protected override bool OnTryLease(int leasePeriodMilliseconds)
             }
             catch (Exception ex)
             {
-                PersistentStorageEventSource.Log.Warning($"Acquiring a lease to file {this.FullPath} has failed.", ex);
+                PersistentStorageEventSource.Log.PersistentStorageException(nameof(FileBlob), $"Acquiring a lease to file {this.FullPath} has failed.", ex);
```

Comment:
How frequent would this happen? and how much perf/memory penalty would this introduce?

Reply:
If this has any chance to be triggered frequently, need to use a heap-alloc-free approach (in other words, no string interpolation).

Reply:
updated - Please review.